app/src/organisms/Devices/ProtocolRun/SetupPipetteCalibrationItem.tsx

```diff
@@ -42,11 +45,13 @@ export function SetupPipetteCalibrationItem({
   mount,
   robotName,
   runId,
-}: SetupPipetteCalibrationItemProps): JSX.Element {
+}: SetupPipetteCalibrationItemProps): JSX.Element | null {
   const { t } = useTranslation(['protocol_setup', 'devices_landing'])
   const deviceDetailsUrl = `/devices/${robotName}`
-
+  const [showFlexPipetteFlow, setShowFlexPipetteFlow] = React.useState(false)
```

Comment:
I think type casing the state setter removes a lint warning

```suggestion
  const [showFlexPipetteFlow, setShowFlexPipetteFlow] = React.useState<boolean>(false)
```